xml/System.Text.Json/JsonSerializer.xml

```diff
@@ -293,7 +293,7 @@ There is remaining data in the span beyond a single JSON value.</exception>
         <typeparam name="TValue">The target type of the JSON value.</typeparam>
         <param name="json">The JSON text to parse.</param>
         <param name="options">Options to control the behavior during parsing.</param>
-        <summary>Parses the text representing a single JSON value into an instance of the type specified by a generic type parameter.</summary>
+        <summary>Parses the text representing a single JSON value into an instance of the type specified by <typeparamref name="TValue" />.</summary>
```

Comment:
All other APIs with generic type parameter `TValue` used the old phrase `a generic type parameter`, i guess they better to stay the same
![image](https://user-images.githubusercontent.com/45579687/95376542-1e377680-0896-11eb-82fd-23560345de08.png)